src/RBControl_manager.cpp

```diff
@@ -189,23 +196,6 @@ bool Manager::motorsFailSafe(void *cookie) {
     return true;
 }
 
-void Manager::initEncoder(MotorId id) {
```

Comment:
Don't we need now this method? The encoder is automatically initialized in `Motor` constructor or the user must use some other method?

Reply:
I have already found the actual [solution](https://github.com/RoboticsBrno/RB3201-RBControl-library/pull/9/files#diff-d219de94c23b39cd5045d2fbc0db8b50R71).

In the new version you firstly check, when you want to get the encoder instance, you firstly check, if is the encoder initialize and when not you initialize it. Am I correct?

Reply:
That is correct.